### PR TITLE
chore(*): fix typos

### PR DESCRIPTION
**Pull request by bbbmj**

### CODE REVIEW

**Comment by grodrigues3** [comment]
/ok-to-test

Why the -f for the push command?  Is that necessary?

File: CLA.md [hunk 1]
```diff
@@ -59,7 +59,7 @@ You must also set your [git e-mail](https://help.github.com/articles/setting-you
 to match this e-mail address as well.
 
 If you've already submitted a PR you can correct your user.name and user.email
-and then use use `git commit --ammend --reset-author` and then `git push` to
+and then use use `git commit --amend --reset-author` and then `git push -f` to
```
Comment (cblecker) [inline CLA.md]: To keep the style consistent, can we use the `--force` flag instead of `-f`?

**Comment by bbbmj** [comment]
fixed @cblecker @grodrigues3 PTAL

File: CLA.md (hunk 1 above)
Comment (bbbmj) [reply CLA.md]: done

**Pull request merged**